simulator/interface/gdb/mips-sim-gdb.cmake

```diff
@@ -0,0 +1,32 @@
+# Build file for MIPT-MIPS GDB interface
+# Vyacheslav Kompan
+# Copyright 2018 MIPT-MIPS
+
+
+set(GDB_SIM_COMMON_PATH ${CMAKE_CURRENT_LIST_DIR}/../../../../sim/common)
+
+# Add mipt-mips-gdb target iff we are building under GDB
+if (EXISTS ${GDB_SIM_COMMON_PATH})
+
+    set(GDB_SIM_SRCS
+            ${GDB_SIM_COMMON_PATH}/sim-utils.c
+            ${GDB_SIM_COMMON_PATH}/callback.c
+            ${GDB_SIM_COMMON_PATH}/sim-io.c)
+
+    add_library(mipt-mips-gdb STATIC ${CPPS}
+            ${CMAKE_CURRENT_LIST_DIR}/gdb_interface.cpp
+            ${GDB_SIM_SRCS})
+
+    # Output: libsim.a stored in "simulator/gdb"
+    set_target_properties(mipt-mips-gdb PROPERTIES
+            PREFIX ""
+            SUFFIX ""
+            OUTPUT_NAME "libsim.a"
+            ARCHIVE_OUTPUT_DIRECTORY "${CMAKE_CURRENT_LIST_DIR}/bin/")
+
+    target_include_directories(mipt-mips-gdb SYSTEM PUBLIC
+            ${CMAKE_CURRENT_LIST_DIR}/../../../..
+            ${CMAKE_CURRENT_LIST_DIR}/../../../../include
```

Comment:
BFD include path is missing.

Reply:
That won't work for nested includes directives:

```bash
In file included from /mnt/d/Dev/miptgdb/sim/common/sim-trace.h:25:0,
                 from /mnt/d/Dev/miptgdb/sim/common/sim-base.h:85,
                 from /mnt/d/Dev/miptgdb/mipt-mips/simulator/interface/gdb/gdb_interface.cpp:18:
/mnt/d/Dev/miptgdb/include/dis-asm.h:35:10: fatal error: bfd.h: No such file or directory
 #include "bfd.h"
          ^~~~~~~
```

Reply:
Looks like it included header from binutils-dev package, so I didn't notice it.